.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+title: ''
+labels: bug report
```

Comment:
Is this label correct? I thought it was only called 'bug' .. also I see in one of the templates, this value is in quotes. Does that matter at all?

Reply:
I have created one